src/ngraph/util.cpp

```diff
@@ -498,3 +498,25 @@ AxisVector ngraph::get_permutation_to_default_order(const AxisVector& axis_order
     }
     return out;
 }
+
+void ngraph::get_pass_enables(std::unordered_map<std::string, bool>& enables)
```

Comment:
Instead of doing it this way, have an enables initialized from the environment and functions for setting/clearing values, and for testing if something is set. That way, unit tests can easily change a value. It might be good to have an RAII save/restore mechanism so a unit test didn't accidentally leave something changed and mess up the remaining tests.

Reply:
@diyessi If a unit-test creates a PassConfig object, how would it pass it to the backend transformer. Do you want me to modify the compile() API to take an additional argument?